src/zenml/integrations/mlflow/steps/mlflow_deployer.py

```diff
@@ -86,23 +86,21 @@ def mlflow_model_deployer_step(
     """
     model_deployer = MLFlowModelDeployer.get_active_model_deployer()
 
-    if not config.model_uri:
-        # fetch the MLflow artifacts logged during the pipeline run
-        experiment_tracker = Repository(  # type: ignore[call-arg]
-            skip_repository_check=True
-        ).active_stack.experiment_tracker
+    # fetch the MLflow artifacts logged during the pipeline run
+    experiment_tracker = Repository(  # type: ignore[call-arg]
```

Comment:
Not too familiar with the entire deployment stuff, but it seems like (at least in this method) `config.model_uri` is not used anymore. Can we get rid of it or does it serve any purpose in different parts of the code?

Reply:
`config.model_uri` may not be used here anymore, but it still needs to be part of the service configuration. It's just set elsewhere (in the model deployer).